newhelm/runners/simple_benchmark_runner.py

```diff
@@ -57,7 +64,7 @@ def run(self, benchmark: BaseBenchmark, suts: List[SUT]) -> List[BenchmarkRecord
             benchmark_records.append(
                 BenchmarkRecord(
                     benchmark_name=benchmark.__class__.__name__,
-                    sut_name=sut.__class__.__name__,
+                    sut_name=sut_name,
                     test_records=test_records,
```

Comment:
This is something intended to be serialized? If I'm trying to figure out how a result ended up the way it did, I'd like to have the class name and any current label.

Reply:
Yes it is intended to be serialized.  I think how much detail we need to serialize evokes a deeper discussion about provenance. For example, if we record the sut name `llama2` and the class used `TogetherCompletionsSUT`, we would need to record all the parameters passed into `TogetherCompletionsSUT` to know if this was `llama-2-7b`, `llama-2-13b`, or `llama-2-70b` just from the record. I see some paths to automatically collecting that info, but it requires making the runner interfaces a little more ugly. Right now figuring that out would require examining the code you used to perform the run.

Lets explore a deeper solution in #127

Reply:
Sounds good.